tests/Tests.ServiceProvider/CustomAutoDataAttributes.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Tests;
+
+[AttributeUsage(AttributeTargets.Method)]
+public class CustomAutoDataAttribute : AutoDataAttribute
+{
+    public CustomAutoDataAttribute() : base(FixtureHelpers.CreateFixture)
+    {
+    }
+}
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public class CustomInlineAutoDataAttribute : InlineAutoDataAttribute
```

Comment:
I would try to stick to the single class per file rule, just so it is easier to find the classes in solution explorer, keep the cognitive load low when I open a file and have better experience when I follow file references (e.g. from stack traces)

Reply:
I generally agree with you. I usually only pack together helper classes like these ones. I feel like too many files for helper methods might distract the onlooker. But thanks for the feedback!